FXyz-Importers/src/main/java/org/fxyz3d/importers/Importer3D.java

```diff
@@ -90,13 +86,15 @@ public static Node load(String fileUrl, boolean asPolygonMesh) throws IOExceptio
      * @return The loaded Node which could be a MeshView or a Group and the Timeline animation
      * @throws IOException if issue loading file
      */
-    public static Pair<Node,Timeline> loadIncludingAnimation(String fileUrl, boolean asPolygonMesh) throws IOException {
+    public static Model loadIncludingAnimation(URL fileUrl, boolean asPolygonMesh) throws IOException {
+        String extForm = fileUrl.toExternalForm();
```

Comment:
Check for possible null URL to avoid NPE (given that URL comes from `getResource` quite often you can get an null resource)